core/domain/summary_services.py

```diff
@@ -339,6 +457,21 @@ def _get_displayable_collection_summary_dicts(collection_summaries):
 def get_library_groups(language_codes):
     """Returns a list of groups for the library index page. Each group has a
     header and a list of dicts representing activity summaries.
+
+    Args:
+        language_codes: list(str). A list of language codes. Filter explorations
+            for their language.
+
+    Return:
+        list(dict). A list of groups for the library index page. Each group is
+        represented by a dict with the following keys and values:
+            - activity_summary_dicts: list(dict). A list of dicts representing
+                activity summaries.
+            - categories: list(str). The list of group categories.
+            - header_i18n_id: str. The i18n id for the header of the category.
+            - has_full_results_page: bool.
```

Comment:
bool. Whether the group header links to a "full results" page. This is always True for the "exploration category" groups.